src/transaction.ts

```diff
@@ -152,6 +153,20 @@ export class Transaction {
     this.hash = TransactionHash.empty();
   }
 
+  /**
+ * Creates a new Transaction object from a DraftTransaction.
+ */
+  static fromDraft(draft: DraftTransaction): Transaction {
+    return new Transaction({
+      sender: Address.fromBech32(draft.sender),
+      receiver: Address.fromBech32(draft.receiver),
+      gasLimit: new BigNumber(draft.gasLimit).toNumber(),
+      chainID: "",
```

Comment:
Yes, this is a bit unfortunate at this moment (since `chainID` is required on the `Transaction`).